languages/tree-sitter-stack-graphs-java/src/stack-graphs.tsg

```diff
@@ -428,13 +1029,46 @@ attribute node_symbol = node            => symbol = (source-text node), source_n
     edge expr_ref__typeof -> expr_ref
 }
 
+(object_creation_expression (primary_expression) @child) @this {
+  edge @child.lexical_scope -> @this.lexical_scope
+}
+
+(object_creation_expression type_arguments: (type_arguments (_) @type)) @this {
+  edge @type.lexical_scope -> @this.lexical_scope
+}
+
+(object_creation_expression type: (_) @type) @this {
+  edge @type.lexical_scope -> @this.lexical_scope
+}
+
+(object_creation_expression arguments: (argument_list (expression) @expr)) @this {
+  edge @expr.lexical_scope -> @this.lexical_scope
+}
```

Comment:
`new` has a fair amount going on.

IMO we should probably have the type point to each of the constructors at a higher priority than to the class declaration.